python/ql/test/experimental/dataflow/typetracking/attribute_tests.py

```diff
@@ -0,0 +1,15 @@
+def simple_read_write():
+    x = object() # $tracked=foo
+    x.foo = tracked # $tracked $tracked=foo
```

Comment:
Good catch! I will rewrite the code so that it actually runs. (I actually had `Foo()` previously, but then rewrote it to `object()` as a bit of clean-up, only I forgot to test the code again afterwards...)